geo/src/main/java/org/odk/collect/geo/SelectionMapFragment.kt

```diff
@@ -57,6 +65,11 @@ class SelectionMapFragment : Fragment() {
 
     private var previousState: Bundle? = null
 
+    @VisibleForTesting
+    internal constructor(selectionMapViewModel: SelectionMapViewModel) : this() {
```

Comment:
Fragments shouldn't have constructors - they are created by the framework. The new `FragmentManager` system actually allows us to add constructors which could possibly give us an alternative to using Dagger. However, the most obvious use at the moment is to create simpler tests that allow us to override dependencies that aren't appropriate for Dagger (like here where we grab the ViewModel from the ViewModelProvider).